provider/testdata/data/baked/mod.rs

```diff
@@ -49,3089 +49,1869 @@ use ::icu_provider::prelude::*;
-                segmenter::lstm::wl_auto_v1::lookup(&req.locale)
-                    .map(zerofrom::ZeroFrom::zero_from)
-                    .map(DataPayload::from_owned)
-                    .map(|payload| DataResponse {
-                        metadata: Default::default(),
-                        payload: Some(payload),
-                    })
-                    .ok_or_else(|| DataErrorKind::MissingLocale.with_req(::icu_segmenter::provider::LstmForWordLineAutoV1Marker::KEY, req))
-            }
-        }
-        #[cfg(feature = "icu_segmenter")]
-        #[clippy::msrv = "1.61"]
-        impl DataProvider<::icu_segmenter::provider::SentenceBreakDataV1Marker> for $provider {
-            fn load(&self, req: DataRequest) -> Result<DataResponse<::icu_segmenter::provider::SentenceBreakDataV1Marker>, DataError> {
-                segmenter::sentence_v1::lookup(&req.locale)
-                    .map(zerofrom::ZeroFrom::zero_from)
-                    .map(DataPayload::from_owned)
-                    .map(|payload| DataResponse {
-                        metadata: Default::default(),
-                        payload: Some(payload),
-                    })
-                    .ok_or_else(|| DataErrorKind::MissingLocale.with_req(::icu_segmenter::provider::SentenceBreakDataV1Marker::KEY, req))
-            }
-        }
-        #[cfg(feature = "icu_segmenter")]
-        #[clippy::msrv = "1.61"]
-        impl DataProvider<::icu_segmenter::provider::WordBreakDataV1Marker> for $provider {
-            fn load(&self, req: DataRequest) -> Result<DataResponse<::icu_segmenter::provider::WordBreakDataV1Marker>, DataError> {
-                segmenter::word_v1::lookup(&req.locale)
-                    .map(zerofrom::ZeroFrom::zero_from)
-                    .map(DataPayload::from_owned)
-                    .map(|payload| DataResponse {
-                        metadata: Default::default(),
-                        payload: Some(payload),
-                    })
-                    .ok_or_else(|| DataErrorKind::MissingLocale.with_req(::icu_segmenter::provider::WordBreakDataV1Marker::KEY, req))
-            }
-        }
-        #[cfg(feature = "icu_timezone")]
-        #[clippy::msrv = "1.61"]
-        impl DataProvider<::icu_timezone::provider::MetazonePeriodV1Marker> for $provider {
-            fn load(&self, req: DataRequest) -> Result<DataResponse<::icu_timezone::provider::MetazonePeriodV1Marker>, DataError> {
-                time_zone::metazone_period_v1::lookup(&req.locale)
-                    .map(zerofrom::ZeroFrom::zero_from)
-                    .map(DataPayload::from_owned)
-                    .map(|payload| DataResponse {
-                        metadata: Default::default(),
-                        payload: Some(payload),
-                    })
-                    .ok_or_else(|| DataErrorKind::MissingLocale.with_req(::icu_timezone::provider::MetazonePeriodV1Marker::KEY, req))
-            }
-        }
+    ($provider:path) => {
```

Comment:
Yeah this is why we moved to `rustfmt`. I think it's a lot more important that this is readable than the `.rs.data` files.